packages/web/components/ProfilePage/index.jsx

```
@@ -46,6 +46,7 @@ const TabContent = ({ tabId, profileId }) => {
           onDeleteConfirmClick={() => { }}
           onEditClick={() => { }}
           onShareNowClick={() => { }}
+          onImageClick={() => { }}
```

Is this to silence an error?

---

I actually don't know why these are here 🙈 I don't think they are doing anything? I tried taking them out and it doesn't seem to make a difference